quantus/helpers/perturb_func.py

```diff
@@ -8,134 +8,160 @@
 import numpy as np
 import scipy
 
-from .utils import get_baseline_value
-from .utils import conv2D_numpy
-
-
-def gaussian_noise(
-    arr: np.array, perturb_mean: float = 0.0, perturb_std: float = 0.01, **kwargs
-) -> np.array:
-    """Add gaussian noise to the input."""
-    noise = np.random.normal(loc=perturb_mean, scale=perturb_std, size=arr.shape)
-    return arr + noise
+from .utils import (
+    get_baseline_value,
+    blur_at_indices,
+    expand_indices,
+    get_leftover_shape,
+)
 
 
 def baseline_replacement_by_indices(
     arr: np.array,
     indices: Union[int, Sequence[int], Tuple[np.array]],
-    for_all_channels: bool = True,
+    indexed_axes: Sequence[int],
+    perturb_baseline: Union[float, int, str, np.array],
     **kwargs,
 ) -> np.array:
     """
-    Replace indices in an array by given baseline.
-    for_all_channels: Replace complete channel for given index.
-                      Assumes channel first ordering.
+    Replace indices in an array by a given baseline.
```

Comment:
Update this docstring to reflect function arguments

Reply:
done

Reply:
missing `perturb_baseline` or is it I who am missing something (hehe)?